bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3ChunkStorage.java

```diff
@@ -65,16 +65,18 @@
     private final S3Client client;
     private final boolean shouldClose;
     private final AtomicBoolean closed;
+    private final boolean supportsAppend;
 
     //endregion
 
     //region constructor
-    public ExtendedS3ChunkStorage(S3Client client, ExtendedS3StorageConfig config, Executor executor, boolean shouldClose) {
+    public ExtendedS3ChunkStorage(S3Client client, ExtendedS3StorageConfig config, Executor executor, boolean supportsAppend, boolean shouldClose) {
```

Comment:
Would it make sense to put `supportsAppend` into the config?

Reply:
For that we have  'storage.appends.enable' in ChunkedSegmentStorageConfig.
But for test purpose we need to control `supportsAppend` separately. I don't want to introduce yet another config value just for that.